modules/tinymce/src/themes/silver/main/ts/ui/statusbar/ResizeHandle.ts

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { AlloyEvents, Behaviour, Dragging, Focusing, NativeEvents, SimpleSpec, SimulatedEvent, Tabstopping } from '@ephox/alloy';
+import { Optional } from '@ephox/katamari';
+import { EventArgs, SugarPosition } from '@ephox/sugar';
+
+import Editor from 'tinymce/core/api/Editor';
+import VK from 'tinymce/core/api/util/VK';
+
+import { UiFactoryBackstageProviders } from '../../backstage/Backstage';
+import { get as getIcon } from '../icons/Icons';
+import { resize, ResizeTypes } from '../sizing/Resize';
+
+const getResizeType = (editor: Editor): ResizeTypes => {
+  // If autoresize is enabled, disable resize
+  const fallback = !editor.hasPlugin('autoresize');
+  const resize = editor.getParam('resize', fallback);
+  if (resize === false) {
+    return ResizeTypes.None;
+  } else if (resize === 'both') {
+    return ResizeTypes.Both;
+  } else {
+    return ResizeTypes.Vertical;
+  }
+};
+
+const keyboardHandler = (editor: Editor, resizeType: ResizeTypes, event: SimulatedEvent<EventArgs<KeyboardEvent>>) => {
```

Comment:
```suggestion
const keyboardHandler = (editor: Editor, resizeType: ResizeTypes, event: NativeSimulatedEvent<KeyboardEvent>) => {
```
This is just an alias for the same type, it's just more readable.